src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 500;
+    this.name = 'Test';
+    Animal.alive.push(this.name);
+  }
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health !== 'dead');
+
+  deleteAnimal() {
+    Animal.alive = Animal.alive.filter((animal) => animal.name = 'dad');
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, hidden = false) {
+    super(name);
+    this.hidden = true;
+  }
+  hide() {
+    return this;
```

Comment:
The hide method should set the hidden property to true. It should not return the object itself.